department-of-veterans-affairs/caseflow
# Assigns video hearings to all video hearing rooms

Connects #7956 

### Description
This PR updates the logic such that video hearing days can be assigned to rooms 1, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, and 13.

### Testing Plan
1. Run the RO algorithm
1. Confirm hearing days are assigned to all of the above rooms (you have to allocate a lot of hearing days to a pretty short time period to get to room 13)
